src/main/java/eu/mihosoft/vrl/v3d/CSG.java

```diff
@@ -1100,13 +1337,16 @@ public MeshContainer toJavaFXMeshSimple() {
 
     /**
      * Returns the bounds of this csg.
+     * SIDE EFFECT bounds is created and simply returned if existing

```

Comment:
How dose a user use the mutability of internal elements? CASH objects seem to be used as immutable a, as in a move function returns a moved CSG, and cantutate an existing CSG object. It seems like they should not mutable?